inference-engine/src/transformations/src/transformations/hswish_fusion.cpp

```diff
@@ -0,0 +1,180 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/hswish_fusion.hpp"
+
+#include <memory>
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+bool check_constant_value(const std::shared_ptr<ngraph::opset4::Constant>& constant,
```

Comment:
Can we unify this function and function  from `swish_fusion.cpp` and move it to utils or some header?

Reply:
If we move this function to utils, then we should make sure it has a unique and self-documenting name.
Something like `is_constant_float` or something similar...

Reply:
I also thought about it, but I wouldn't like to touch the Swish op files in this PR. I can create a separate PR with enabling this function as a helper, after this one will be merged.

Reply:
@mitruska ok